src/@commonsku/styles/Pagination.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import styled from "styled-components";
+import { ChevronIcon } from "./icons";
+import colors from "./colors";
+
+//Styles Begin
+const ScrollControl = styled.tr`
```

Comment:
This should probably just be a generic `div` rather than a `tr`, since it's not (necessarily) in a table.